spacy/tests/pipeline/test_models.py

```diff
@@ -0,0 +1,106 @@
+from typing import List
+
+import numpy
+import pytest
+from numpy.testing import assert_almost_equal
+from spacy.vocab import Vocab
+from thinc.api import get_current_ops, Model, data_validation
+from thinc.types import Array2d, Ragged
+
+from spacy.lang.en import English
+from spacy.ml import FeatureExtractor, StaticVectors
+from spacy.ml._character_embed import CharacterEmbed
+from spacy.tokens import Doc
+
+OPS = get_current_ops()
```

Comment:
Doing `get_current_ops()` in the global scope is probably bad, because you're either going to get the default (NumpyOps) or maybe somewhere else is setting CupyOps as the default and then it comes down to an import race.

So, I would either move this into a function, or just make a `NumpyOps` instance. Either one is fine.

Reply:
This should probably just be a `pytest.fixture`?